lib/matplotlib/collections.py

```diff
@@ -117,7 +122,10 @@ def __init__(self,
         """
         artist.Artist.__init__(self)
         cm.ScalarMappable.__init__(self, norm, cmap)
-
+        self._us_linestyles = [(None, None)]
```

Comment:
FWIW, it was not obvious to me that “_us” prefix stood for “unscaled” (at least until I read ~ l. 390 in `lines.py`). Adding a small comment might be useful for future readers.